suites/reef/rgw/tier-2_rgw_ms_archive_with_haproxy.yaml

```diff
@@ -653,3 +653,64 @@ tests:
       polarion-id: CEPH-83575473
       module: sanity_rgw_multisite.py
       name: test rgw restore index tool on versioned bucket
+  - test:
+      clusters:
+        ceph-pri:
+          config:
+            cephadm: true
+            commands:
+              - "ceph config set client.rgw.shared.pri rgw_crypt_default_encryption_key 4YSmvJtBv0aZ7geVgAsdpRnLBEwWSWlMIGnRS8a9TSA="
```

Comment:
@viduship please add this config in pacific suite also

Reply:
done @hmaheswa